clarinet.js

```diff
@@ -70,6 +72,93 @@ else env = window;
   // switcharoo
   S = clarinet.STATE;
 
+  const Char = {
```

Comment:
I have a hard time envisioning a future where letter and number constants have any use.  Perhaps trim this down to just the codes used in this PR?